### DISCUSSION TITLE
How many people in the room? 👨‍💼👩‍💼

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1224] [Galaxy Trivia] [answered]
Alice is late for a meeting. After she enters the room, the average age of everyone in the room increases by exactly 4 years. A few minutes later, her twin sister Sarah enters the meeting room and the average age of everyone in the room increases by exactly 3 years.

**Comment by Camarota-234** [comment]
There are 8 people in the room by the end of the events, meaning that there were 6 people in the meeting before Alice entered.

**Comment by peterbaccaro** [comment, marked as answer]
There are **6** people in the room.

**Reasoning**

**N** - the original number of people in the room.
**A** - original average age of the N people in the room.
**T**  - the age of each twin.

a) The original total weight in the room is **NA**.
b) After the first twin enters the total weight increases by: **(N+1)(A+4)-NA = T** which simplified is **4N+A+4 = T**
c) After the second twin enters the total weight increases from original by: **(N+2)(A+7)-NA = 2T** which simplified is **7N+2A+14 = 2T**

Now we compute 2 times b) minus c) and get N=6 as suggested at the start.

**Comment by PasinduUpendra** [comment]
Assume average age is 0 for simplicity.


``` 
 8(x+1)=7(x+2)
 8x+8=7x+14
 x=14−8
 x=6
```

**Comment by danielcshn** [comment]
Alice age is twice Sarah age, but it doesn't directly provide the number of people.

**Comment by cdprf** [comment]
if there was 2 people, alice came  and assume that 1 of the room is 1 year old and other one is only 0 year old..

1+0=1/2=0  (i can round it where ever i want , upside or down :);

alice was 7 years old.  so  10 + 1+0= 11  /  3 = avg is 4  

when the other betch  arrive to room. she 8 years old too  10 + 11= 21  /4 = avg is 5
